cifar10_tutorial.ipynb

```diff
@@ -207,7 +207,7 @@
     "    def forward(self, x):\n",
     "        x = F.max_pool2d(F.relu(self.conv1(x)), 2)\n",
     "        x = F.max_pool2d(F.relu(self.conv2(x)), 2)\n",
-    "        x = x.view(-1, 16 * 5 * 5)\n",
+    "        x = x.flatten(start_dim=1)\n",
```

Comment:
I think for now on, the recommended approach for this kind of flattening is to use `.flatten`, as it is simpler to use and fits well with the use-cases that we have for going from convolutional feature maps to 2d tensors